### PR TITLE
Oraclejava

### PR DESCRIPTION
**Pull request by kayiwa**
testing this on travis has been a bear when it comes to determining what processor to use. Will continue to investigate our to add integration tests for java.

### CODE REVIEW

**Comment by kayiwa** [comment]
ping! OK?

File: roles/pulibrary.openjdk8/molecule/default/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: instance
+    image: centos:7
```
Comment (jrgriffiniii) [inline roles/pulibrary.openjdk8/molecule/default/molecule.yml]: Why is the CentOS image being used here?

File: roles/pulibrary.oracle.java8/molecule/default/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: instance
+    image: centos:7
```
Comment (jrgriffiniii) [inline roles/pulibrary.oracle.java8/molecule/default/molecule.yml]: Please see the above question regarding CentOS

**Comment by jrgriffiniii** [comment]
Sorry for the delay.  This is excellent, I only had two questions regarding the usage of CentOS Docker images.

Also, would adding any server-specific JDK configuration options be possible a future PR?  If so, I can create an issue following merging this.

File: roles/pulibrary.oracle.java8/molecule/default/molecule.yml (hunk 2 above)
Comment (kayiwa) [reply roles/pulibrary.oracle.java8/molecule/default/molecule.yml]: the way the repo is set up, we never really use this. we rely `molecule/default/molecule.yml` from the repo root.

**Pull request merged**